op-node/p2p/peer_scorer_test.go

```diff
@@ -92,17 +101,23 @@ func (testSuite *PeerScorerTestSuite) TestSnapshotHookBlockPeer() {
 		testSuite.mockGater,
 		testSuite.mockStore,
 		testSuite.mockMetricer,
+		testSuite.mockBandScorer,
 		testSuite.logger,
 	)
 	inspectFn := scorer.SnapshotHook()
 
-	// Mock the snapshot updates
-	// This doesn't return anything
-	testSuite.mockMetricer.On("RecordPeerScoring", peer.ID("peer1"), float64(-101)).Return(nil)
+	// Mock the band scorer calls
+	testSuite.mockBandScorer.On("Reset").Return(nil)
+	testSuite.mockBandScorer.On("Bucket", float64(-101)).Return("graylist")
```

Comment:
If you used the actual `bandScorer` here would you get the same results?

Reply:
Yes - Will remove the mock to trim down the pr :)